pandas/tools/tests/test_plotting.py

```diff
@@ -0,0 +1,20 @@
+from numpy.testing import assert_equal
+import pandas.util.testing as tm
+import matplotlib as mpl
+import pandas as pd
+
+class TestStandardColors(tm.TestCase):
+    def test_plot(self):
+        color_tuples = [(0.9, 0, 0, 1), (0, 0.9, 0, 1), (0, 0, 0.9, 1)] 
+        colormap = mpl.colors.ListedColormap(color_tuples)
+        barplot = pd.DataFrame([[1,2,3]]).plot(kind="bar", cmap=colormap)
+        assert_equal(color_tuples, [c.get_facecolor() for c in barplot.patches])
+
+    def test_rcParams(self):
+        color_tuples = [(0.9, 0, 0, 1), (0, 0.9, 0, 1), (0, 0, 0.9, 1)]
+        try:
+            mpl.rcParams["axes.prop_cycle"] = mpl.cycler("color", color_tuples)
```

Comment:
We'll need to be careful not to modify rcParams for the rest of the tests. Can't look right now, but I think we or matplotlib have a contextmanager that handles this. @sinhrks do you remember? Also you can move this to `tpandas/tools/tests/test_graphics.py` and add it to one of the test classes in there.